lib/core/presentation/pages/chat/chat_message/chat_message_page.dart

```diff
@@ -138,7 +136,7 @@ class ChatController extends State<ChatPageWithRoom> {
     if (!timeline!.canRequestHistory) return;
     Logs().v('Requesting history...');
     try {
-      await timeline?.requestHistory(historyCount: _loadHistoryCount);
+      await timeline?.requestHistory();
```

Comment:
For better performance, fetch default list for history and future data instead of 100